OsmAnd-java/src/net/osmand/binary/BinaryMapAddressReaderAdapter.java

```diff
@@ -593,6 +594,7 @@ protected Building readBuilding(int fileOffset, int street24X, int street24Y, Li
 	}
 
 	public void searchAddressDataByName(AddressRegion reg, SearchRequest<MapObject> req, List<Integer> typeFilter) throws IOException {
+		Set<LatLon> added = new HashSet<>();
```

Comment:
Maybe the name "added" can be a little more explicit.
What does it really means?
(it contains the list of LatLong that were already found ?)